src/app/shared/navbar/navbar.html

```diff
@@ -1,6 +1,11 @@
 <!-- TODO: figure out if the <nav> should go inside of a <header> element. -->
 <nav class="docs-navbar-header" aria-label="Top Toolbar"
      [class.is-next-version]="isNextVersion">
+  <div class="skip-link-wrapper" [class.cdk-visually-hidden]=skipLinkHidden>
```

Comment:
Style nit: always use quotes for the attribute values, i.e. `[class.cdk-visually-hidden]="skipLinkHidden"` and `[href]="skipLinkTarget"`